sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -16,6 +16,7 @@
 from .opensearch_utils.document import Document
```

Comment:
Where does this indexer run?

Reply:
It runs on a lambda. This lambda get's kickoff after we drop a file on data bucket.

Reply:
Got it, thanks! That was my guess, just making sure.